### PR TITLE
Add the Lucky framework. #270

### PR DESCRIPTION
**Pull request by gabriel-arauj**

### CODE REVIEW

File: images/svg/luckyframework.svg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" 
+aria-label="Lucky framework" viewBox="0 0 512 512">
```
Comment (edent) [inline images/svg/luckyframework.svg]: Needs `role="img"`

File: images/svg/luckyframework.svg [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+<rect width="512" height="512" rx="15%" fill="#fff"/>
+<path fill="none" d="M0 0h512v512H0z"/>
+<path d="M149.596 402.483C188.79 266.35 197.171 279.003 294.4 78.968c26.415 12.515 26.728 12.963 98.75 141.148 28.475 50.683 42.263 77.142 43.82 84.097 9.22 50.237-24.668 98.992-60.886 102.18-75.535.812-147.627-.3-226.488-3.91z" fill="#9cfd65"/>
```
Comment (edent) [inline images/svg/luckyframework.svg]: You can remove these newlines to save a few bytes.

File: images/svg/luckyframework.svg (hunk 1 above)
Comment (gabriel-arauj) [reply images/svg/luckyframework.svg]: Ok. Solved.

File: images/svg/luckyframework.svg (hunk 2 above)
Comment (gabriel-arauj) [reply images/svg/luckyframework.svg]: Nice.

Comment (gabriel-arauj) [reply images/svg/luckyframework.svg]: Solved.

**Comment by gabriel-arauj** [comment]
@edent, problems solved.

**Comment by edent** [comment]
Woo!

**Comment by gabriel-arauj** [comment]
:o 
I didn't know this was possible

**Pull request merged**